### ISSUE TITLE
FBAudienceNetwork warnings

### ISSUE THREAD

**Issue by levochkaa** [cleveradssolutions/CAS-iOS#6]
FBAudienceNetwork had an issue in 6.15.0, https://github.com/facebook/facebook-ios-sdk/issues/2390.
![image](https://github.com/cleveradssolutions/CAS-iOS/assets/62802017/ce9eb4c8-515d-4272-9ec1-e908d1cc0b5f)

They seemed to fix it in 6.15.1, https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/changelog#6_15_1

**Comment by Str4tos** [comment]
Hi @levochkaa, thank you for raising the issue.  
We are aware of this issue and are preparing to update the Audience Network Framework version. This update also requires the use of Xcode 15.3. 
We are delaying the update because we do not consider the issue critical, but please correct us if this is not the case.

**Comment by levochkaa** [comment]
Hi @Str4tos, shortly: this isn't a critical issue for us, take your time for the update.

I'm just preparing our project for migration to Xcode 16, when it releases. So, when moving from Xcode 15.4 to 16 (beta 2-3), I get this error (on 3.3.2 version):
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_ADClient", referenced from:
       in MyTrackerSDK[arm64][78](MRIadHandler.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Updating to 3.6.1 and higher fixes all build errors, but brings these warnings :)

**Comment by Str4tos** [comment]
Fixed in CAS 3.9.0 update.

**Comment by levochkaa** [comment]
Thank you!
